### PR TITLE
dwm-status: add as user service

### PR DESCRIPTION
**Pull request by Gerschtli**
Hey, I have added a service module for my own package called [`dwm-status`](https://github.com/Gerschtli/dwm-status). It is already present in nixpkgs for a long time now.

### CODE REVIEW

**Comment by Gerschtli** [comment]
Thank you for the comments!

File: modules/services/dwm-status.nix [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  configFile = pkgs.writeText "dwm-status.json" configText;
+in
+
+{
+  options = {
+    services.dwm-status = {
+      enable = mkEnableOption "dwm-status user service";
+
+      package = mkOption {
```
Comment (rycee) [inline modules/services/dwm-status.nix]: Was just about to merge this but started thinking about the `enableAlsaUtils` and `enableNetwork` parameters. Would it not be possible to have these be actual options (of `bool` type) in this module? One could then have
```
ExecStart =
  let
    package = pkgs.dwm-status.override {
      inherit (cfg) enableAlsaUtils enableNetwork;
    }
  in
    "${package}/bin/dwm-status ${configFile}";
```

and simply remove the `services.dwm-status.package` option.

Comment (Gerschtli) [reply modules/services/dwm-status.nix]: I had a similar discussion in my PR in nixpkgs. These two package overrides are for users who want minimal closure size. If you disable one of these options, you need to recompile the complete rust binary, which is unreasonable for the average user. Without providing these as separate option, the temptation is not that big to use these options.

Comment (rycee) [reply modules/services/dwm-status.nix]: Ok, fair enough. I'll merge as-is 🙂

But if this is only for extreme cases then I would suggest simply removing this option and letting sufficiently interested users override the dwm-status attribute using an overlay or `nixpkgs.config.packageOverrides`.

**Comment by rycee** [comment]
Rebased to master in db0dfb4b089d0f5ba86ab39b40a228ae47c5d440 👍

File: modules/services/dwm-status.nix (hunk 1 above)
Comment (Gerschtli) [reply modules/services/dwm-status.nix]: I tried to be consistent with most modules in nixpkgs with the `package` option.

**Pull request closed without merging**